python/ray/_private/ray_constants.py

```diff
@@ -124,6 +124,10 @@ def env_bool(key, default):
 # Number of units 1 resource can be subdivided into.
 MIN_RESOURCE_GRANULARITY = 0.0001
 
+# Set this environment variable to populate the dashboard URL with
+# and external hosted Ray dashboard URL.
```

Comment:
I'd suggest replacing this with

```
Set this environment variable to populate the dashboard URL with and external Ray dashboard URL (e.g. because the dashboard is behind a proxy or load balancer).
```